_posts/2017-08-08-CompetitiveCoding.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Competitive Coding: From Novice to Expert"
```

Comment:
I feel the title is misleading.
You should link other competitive programming blogs.
Such as 
https://www.hackerearth.com/practice/notes/getting-started-with-the-sport-of-programming/
By Triveni Mahatha ,IIT Kanpur